commands/visualstudio.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+var Q = require('q');
+
+var lib = require('manifoldjs-lib');
+
+var CustomError = lib.CustomError,
+    exec = lib.processTools.exec,
+    fileTools = lib.fileTools,
+    log = lib.log;
+
+var open = require('./open');
+
+function isWindows10Version (version) {
+  return /^10/.test(version);
+}
+
+function getWindowsVersion (callback) {

```

Comment:
This method is a bit confusing. Maybe just get it to return a Q.deferred for the result on success, or would that break things? It seems strange that is sometimes returns a rejected promise, otherwise it returns nothing an uses the callback.